src/shared/netif-naming-scheme.h

```diff
@@ -1,6 +1,7 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 #pragma once
 
+#include "sd-device.h"
```

Comment:
Our own headers should be after standard (and library) headers.

Reply:
fixed